README.md

```diff
@@ -449,27 +449,33 @@ This updates core `logdize_logger` DB function. No need to update tables or trig
 If you want to update Logidze settings for the model, run migration with `--update` flag:
 
 ```sh
-rails generate logidze:model Post --update --only=title,body,rating
+bundle exec rails generate logidze:model Post --update --only=title,body,rating
 ```
 
 You can also use the `--name` option to specify the migration name to avoid duplicate migration names:
 
 ```sh
-$ rails generate logidze:model Post --update --only=title,body,rating --name add_only_filter_to_posts_log_data
+$ bundle exec rails generate logidze:model Post --update --only=title,body,rating --name add_only_filter_to_posts_log_data
 
     create db/migrate/20202309142344_add_only_filter_to_posts_log_data.rb
 ```
 
+### Pending upgrade check
+
+By default, Logidze raises an error when an upgrade is needed. To use warning instead of raising an error set `Logidze.raise_on_pending_upgrade = false`.
+
+If you want to skip the check entirely, use `Logidze.check_pending_upgrade = false`.
```

Comment:
Can we combine these options into a single one? `Logidze.on_pending_upgrade = :raise | :warn | :ignore`

Reply:
Good idea, I'll add a check for the whitelisted options also then.